hmda-spark-reporting/src/main/scala/com/hmda/reports/processing/AggregateProcessing.scala

```diff
@@ -279,8 +351,10 @@ object AggregateProcessing {
     }
 
     val result = for {
-            _ <- persistJson(aggregateTable1)
-            _ <- persistJson2(aggregateTable2)
+      _ <- persistJson(aggregateTable1)
+      _ <- persistJson2(aggregateTable2)
+      _ <- persistJsonI(aggregateTableI.toList)
```

Comment:
`persistJsonI(aggregateTableI.toList)` is being performed twice here. Is that intended?